### PR TITLE
Make documentation HTML renderable

### PR DESCRIPTION
**Pull request by Nadahar**
Hi.

Thanks for making this great plugin. I just made this little PR as a "suggestion" as it makes the documentation easier to read from GitHub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ This plugin is included in my unofficial [NSIS builds](https://github.com/negrut
 - Supports files larger than 4GB
 - Can download remote content directly to Memory (NSIS string) instead of file
 - Works well in **64-bit** [NSIS builds](https://github.com/negrutiu/nsis)
-- A lot more... Check out the included [documentation](NScurl.Readme.htm)
+- A lot more... Check out the included [documentation](https://rawcdn.githack.com/negrutiu/nsis-nscurl/6c9f7c02b050f5db62963d6a9962d6b791e4c7a4/NScurl.Readme.htm)
```
Comment (negrutiu) [inline README.md]:
```suggestion
- A lot more... Check out the included [documentation](https://raw.githack.com/negrutiu/nsis-nscurl/master/NScurl.Readme.htm)
```

File: README.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ Pop $0
 NScurl::http POST "https://httpbin.org/post" Memory /POST "User" "My user name" /POST "Password" "My password" /POST FILENAME=maiden.json TYPE=application/json "Details" '{ "number_of_the_beast" : 666 }' /END
 Pop $0
 ```
-- More complex examples in the [documentation](NScurl.Readme.htm)
+- More complex examples in the [documentation](https://rawcdn.githack.com/negrutiu/nsis-nscurl/6c9f7c02b050f5db62963d6a9962d6b791e4c7a4/NScurl.Readme.htm)
```
Comment (negrutiu) [inline README.md]:
```suggestion
- More complex examples in the [documentation](https://raw.githack.com/negrutiu/nsis-nscurl/master/NScurl.Readme.htm)
```

**Review by negrutiu — CHANGES_REQUESTED** [review]
Great suggestion, thank you.
How about changing the link to point to the latest file version (instead of a specific commit)?

**Comment by Nadahar** [comment]
The reason I didn't point to the branch/master/main is what they say here: http://raw.githack.com/

I don't fully understand what they do, but they somehow cache the result so that it won't be dynamically updated. That said, they do say that you can link to a branch "for development", but that they will throttle traffic for these. "For development" and "in production" isn't defined any further, so it's kind of hard to know what to consider this. I doubt this will cause any significant traffic. I'd say that choice should be up to you 😉 

Edit: Git tags can be used instead of commit hashes, but that doesn't really help as it still requires these to be updated when the documentation changes.

**Review by negrutiu — APPROVED** [review]

**Pull request merged**